shaded/cosn/pom.xml

```diff
@@ -0,0 +1,193 @@
+<!--
+
+    The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+    (the "License"). You may not use this work except in compliance with the License, which is
+    available at www.apache.org/licenses/LICENSE-2.0
+
+    This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+    either express or implied, as more fully set forth in the License.
+
+    See the NOTICE file distributed with this work for information regarding copyright ownership.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.alluxio</groupId>
+    <artifactId>alluxio-shaded</artifactId>
+    <version>2.5.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>alluxio-shaded-cosn</artifactId>
+  <packaging>jar</packaging>
+  <name>Alluxio Shaded Libraries - Cosn</name>
+  <description>Shaded Cosn Module</description>
+
+  <properties>
+    <build.path>${project.parent.parent.basedir}/build</build.path>
+    <cosn.hadoop-cos.version>3.1.0-5.8.5</cosn.hadoop-cos.version>
+    <cosn.cos_api.version>5.6.19</cosn.cos_api.version>
+    <cosn.hadoop.version>3.1.0</cosn.hadoop.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-api</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-core</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-slf4j-impl</artifactId>
+      <scope>provided</scope>
+    </dependency>
+  </dependencies>
+
+  <profiles>
```

Comment:
As now, we have only one profile, and it is active by default, you can put these dependencies directly, when you need to define another profile, we can bring this back.